dotnet/src/SemanticKernel.UnitTests/Contents/AnnotationContentTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using Microsoft.SemanticKernel.Agents.OpenAI;
+using Xunit;
+
+namespace SemanticKernel.UnitTests.Contents;
+
+#pragma warning disable SKEXP0110
+
+/// <summary>
+/// Unit testing of <see cref="AnnotationContent"/>.
+/// </summary>
+public class AnnotationContentTests
```

Comment:
Since polymorphic serialization is enabled for `AnnotationContent` class, should we add tests for serialization/deserialization? Same for `FileReferenceContent` class. 
cc: @SergeyMenshykh

Reply:
Good point.  Will address.

Reply:
Added tests to `ChatMessageContentTests.ItCanBeSerializeAndDeserialized`